README.md

```diff
@@ -1,14 +1,61 @@
-# Time  Series Analysis Suite
+# Deep4cast 
 
-## Forecasting
-We plan to develop an easy to use multivariate TS forecasting software framework, based on LSTM and with confidence estimate.
+***This package is under development.***
 
+```Deep4cast``` provides easy access to deep learning-based tools for forecasting, anomaly detection, and counterfactual analysis.
 
-## Anomaly Detection
+### Forecasting
+Quickstart tutorial can be found in ```/tutorials/quickstart.ipynb```.
 
-- **[Demo](http://nbviewer.jupyter.org/github/MSRDL/TSA/blob/master/demo.ipynb)**
-- **[Presentation: Streaming Least Squares Algorithm for Anomaly Detection](https://1drv.ms/p/s!Ar57GyE1TzYYh_UGeQ8ATUAytx52Fw)**
+### Anomaly Detection
+Anomaly detection turorial coming soon.
 
+### Counterfactual Analysis
+Counterfactual analysis tutorial coming soon.
 
+## Documentation
+Documentation can be found in ```/docs/build/html/index.html```.
+
+## Installation
+### Anaconda
```

Comment:
While Anaconda is convenient to most, this package shouldn't have any dependency on the big fat Anaconda.

Reply:
Will do.